### PR TITLE
Bugfix/share price history

### PR DESCRIPTION
**Pull request by mrhatbirb**
- Removes Share Price tab in TVL dialog.
- Adjusts hover chart label in dark mode
- Minor UI/Linting fixes

### CODE REVIEW

File: src/app/dialogs/download-historic-data-dialog/download-historic-data-dialog.component.ts [hunk 1]
```diff
@@ -11,7 +11,7 @@ import {environment} from '../../../environments/environment';
 export class DownloadHistoricDataDialogComponent implements OnInit {
   sortedVaults: string[];
   includeInactive = false;
-  apiEndpoint= "";
+  apiEndpoint = '';
```
Comment (cryptopower80) [inline src/app/dialogs/download-historic-data-dialog/download-historic-data-dialog.component.ts:14]: is this expected?

File: src/app/dialogs/tvl-dialog/tvl-dialog.component.html [hunk 2]
```diff
@@ -9,40 +9,23 @@
...
-          {{data.type}} Share price history
-        </div>
-        <div #chart class="tvl-chart"></div>
-
-      </mat-tab>
-
-    </mat-tab-group>
-
+    <div *ngIf="data.type !== 'income'">
+
```
Comment (cryptopower80) [inline src/app/dialogs/tvl-dialog/tvl-dialog.component.html:20]: can you please remove this unnecessary space?

File: src/scss/themes/theme-classes.scss [hunk 3]
```diff
@@ -30,29 +29,22 @@ $variables in $themes_colors {
                 color: lighten(map-get($variables, "links"), 5%);
             }
         }
+
+        .mat-tab-label, .mat-tab-link {
+            color: inherit!important;
+        }
+
```
Comment (cryptopower80) [inline src/scss/themes/theme-classes.scss:36]: same comment as above

File: src/app/dialogs/download-historic-data-dialog/download-historic-data-dialog.component.ts (hunk 1 above)
Comment (mrhatbirb) [reply src/app/dialogs/download-historic-data-dialog/download-historic-data-dialog.component.ts:14]: Yes. Our ESLint configuration/guidelines use single quotes and it marks this as a listing error.

File: src/app/dialogs/tvl-dialog/tvl-dialog.component.html (hunk 2 above)
Comment (mrhatbirb) [reply src/app/dialogs/tvl-dialog/tvl-dialog.component.html]: Done :)

File: src/scss/themes/theme-classes.scss (hunk 3 above)
Comment (mrhatbirb) [reply src/scss/themes/theme-classes.scss]: Done~

**Review by cryptopower80 — APPROVED** [review]

**Pull request merged**